solution/backend/regulations/tests/test_statute_links.py

```diff
@@ -134,3 +137,55 @@ def test_link_statutes(self):
             template = Template("{% load link_statutes %}{{ paragraph|link_statutes:link_conversions|safe }}")
             context = Context({"paragraph": test["input"], "link_conversions": link_conversions})
             self.assertEqual(template.render(context), test["expected"], f"Failed while testing {test['testing']}.")
+
+
+class StatuteConvertersAPITestCase(APITestCase):
+    objects = [
+        {
+            "section": "10211",
+            "title": 42,
+            "usc": "18201",
+            "act": "Affordable Care Act",
+            "name": "Definitions.",
+            "statute_title": "X",
+            "source_url": "https://test.gov/test.xml",
+        },
+        {
+            "section": "11111",
+            "title": 45,
+            "usc": "12345",
+            "act": "Social Security Act",
+            "name": "Something",
+            "statute_title": "Y",
+            "source_url": "https://test.gov/test2.xml",
+        },
+        {
+            "section": "7890",
+            "title": 42,
+            "usc": "77777",
+            "act": "Social Security Act",
+            "name": "Something else.",
+            "statute_title": "Z",
+            "source_url": "https://test.gov/test3.xml",
+        },
```

Comment:
not a flag but maybe we should as a team discuss using fixture files for larger json objects.  If its like one object within the list its typically fine but when you have multiple it might make sense to just create a fixture file to be called upon by other functions if needed or even our other types of test like cypress or jest.

Reply:
Addressed